README.md

```diff
@@ -145,14 +145,20 @@ Feel free to leave your feedback in form of [GitHub issues](https://github.com/S
 Want to contribute? Check out our [contribution](./CONTRIBUTING.md) guide and follow our [code of conduct](./CODE_OF_CONDUCT).
 
 ## Links
-
-- [Documentation portal](https://sap.github.io/cloud-sdk/)
+- [Official support channel](https://github.com/SAP/cloud-sdk-js/issues/new/choose)
+  <br>
+  <br>
+- [Github](https://github.com/SAP/cloud-sdk-js)
+  <br>
+  <br>
+- [SAP Cloud SDK Documentation portal](https://sap.github.io/cloud-sdk)
+- [SAP Cloud SDK Documentation portal - Getting started guide](https://sap.github.io/cloud-sdk/docs/js/getting-started)
+- [SAP Cloud SDK Documentation portal - API documentation](https://sap.github.io/cloud-sdk/api/latest)
+- [SAP Cloud SDK Documentation portal - Error handling](https://sap.github.io/cloud-sdk/docs/js/features/error-handling)
+- [SAP Cloud SDK Documentation portal - Release notes](https://sap.github.io/cloud-sdk/docs/js/release-notes)
+  <br>
+  <br>
```

Comment:
```suggestion
- [Official support channel](https://github.com/SAP/cloud-sdk-js/issues/new/choose)
- [Github](https://github.com/SAP/cloud-sdk-js)
- [SAP Cloud SDK Documentation portal](https://sap.github.io/cloud-sdk)
  - [SAP Cloud SDK Documentation portal - Getting started guide](https://sap.github.io/cloud-sdk/docs/js/getting-started)
  - [SAP Cloud SDK Documentation portal - API documentation](https://sap.github.io/cloud-sdk/api/latest)
  - [SAP Cloud SDK Documentation portal - Error handling](https://sap.github.io/cloud-sdk/docs/js/features/error-handling)
  - [SAP Cloud SDK Documentation portal - Release notes](https://sap.github.io/cloud-sdk/docs/js/release-notes)
```
[req] Like Florian mentioned, the <br>'s seem to not work so well. I also personally don't like the look. Indentation may be a good alternative.